BepInEx.IL2CPP/BepInEx.IL2CPP.csproj

```diff
@@ -53,11 +49,5 @@
     <PackageReference Include="Iced" Version="1.6.0" />
     <PackageReference Include="MonoMod.RuntimeDetour" Version="20.11.5.1" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Properties" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Remove="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
```

Comment:
How is this related to semver change? This should be in a separate PR for better testability and ability to revert it if needed.